instance.go

```diff
@@ -380,3 +391,15 @@ func (c *Client) SetInstanceFirewall(id, firewallID string) (*SimpleResponse, er
 	response, err := c.DecodeSimpleResponse(resp)
 	return response, err
 }
+
+// GetInstanceVNCDetails gets the VNC details for an instance
+func (c *Client) GetInstanceVNCDetails(id string) (*InstanceVNCDetails, error) {
```

Comment:
May we name it like `InitInstanceVNC`?
Because it's a PUT and not really a get

Reply:
Makes more sense now.
I was intrigued by the naming as it felt like it's an API that gives you the VNC details. (A fetch API hence the naming as GET)
But I guess in backend, it initiates -> then gives back the result I guess.
I will make these changes.

Reply:
Addressed the comment 👍